fud/fud/stages/verilator/fixed_point.py

```diff
@@ -12,81 +15,178 @@ def binary_to_base10(bitstring: str) -> int:
     return out
 
 
-def fp_to_decimal(value, width, int_width, is_signed):
-    """Takes in a fixed point number with the
-    bit width, integer bit width, and signedness, and
-    returns the decimal value."""
-    frac_width = width - int_width
-    begin_index = 1 if is_signed else 0
+def negate_twos_complement(bitstring: str, width: int) -> str:
```

Comment:
what happens if the bitstring is not representable in `width` bits. throw an error or maybe just calculate width as `len(bitstring)`

Reply:
Good call. I forget why I even included width here.